docs/WalletGuide.md

```diff
@@ -490,7 +541,7 @@ suspend fun main() {
 Build on and off ramps with anchors for deposits and withdrawals.
 
 ```kotlin
-val anchor = wallet.anchor("testanchor.stellar.org")
+val anchor = wallet.anchor(Url("https://testanchor.stellar.org"))
```

Comment:
This seems like something we can do inside the `anchor()` function. Why do we need to have the user wrap the URL string in the `Url` class?

Reply:
I'm still thinking on how to organize it better. 
The thing is that we need to unify links. The easiest way to do it is to have Url type. Because when it's string, it's unclear if it's just domain or if it's full path .
The other way that I used in previous commit was
```kotlin
anchor({ host = "testanchor.stellar.org" })
```
Not sure if it's better or not

Reply:
I guess the best option would be to accept string and try to understand if it's full path or just a host. 
Though, if user passes something unexpected, there could be some obscure errors

Reply:
Stellar uses the concept of home domains in a lot of different places. Stellar account's have a `home_domain` attribute, which points too the domain hosting your SEP-1 toml file. Home domains are also passed in SEP-10 for client attribution. I think we should try to continue using domains in the Wallet SDK to keep this pattern since it's already familiar.

Reply:
I will do it like this. It'll be allowed to pass either home domain or full url

Reply:
It accepts string again (in either format)